DockerfileDev

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
+
+RUN apt-get update --yes && apt-get upgrade --yes
+RUN apt-get install --yes \
+	apt-utils \
+	build-essential \
+	debhelper \
+	devscripts \
+	make \
+	python3 \
+	python3-dev \
+	python3-virtualenv \
+	sudo \
+	virtualenv \
+	nodejs \
+	npm
+
+COPY . /app
+WORKDIR /app
+
+RUN make devel
+
+EXPOSE 8001
```

Comment:
What kind of problem we're trying to solve here?

Reply:
I want to expose the server mentioned [here](https://github.com/ezaquarii/vpn-at-home#for-development) to the host machine. That server is exposed on port 8001.

Reply:
`make runserver` starts Django development server only and by default it runs on 8001 as it is convenient for me - I run a prod version on port 8000.

If you want to run Django `runserver`, just call `./bin/manage runserver --insecure 8001` instead of `make runserver`.
You can also run daphne. Check out `./bin/daphne.sh --help` for details.

Reply:
I'm still struggling to understand why we want to run development version in Docker. The whole point of having a dev environment is that it runs in your IDE, allowing you to write code and run debugger. I could be missing something here.

Reply:
I'll update the PR description in a minute

Reply:
Is there any way to intergrate this into Dockerfile and keep only one file around? Ideally I'd like to "inherit" builder config to maintain only one dependencies installation procedure.

Reply:
In theory yes. Usually there would be just one Dockerfile. Then you would just indicate via env vars if you're in development mode and based on that you could f.e. populate the DB or similar.

In this case I don't know, since I am not very familiar what all is entailed by the current production built that is done.

I think it should be possible, but might be better to pursue in an subsequent PR.

Reply:
Ok, let's work on this after landing this PR.